sql/queries.sql

```diff
@@ -362,3 +362,236 @@ CREATE VIEW user_balance
+WHERE employer_id=11;
+
+-- -----------------------------------------------------------------------
+#ii 
+-- CREATING A CATEGORY BY AN EMPLOYER
+INSERT INTO employer (employer_id, employer_name, description, employer_tier)
+	VALUES (6, 'Mike', 'Front_end developer', '0');
+INSERT INTO employer_category(employer_id, category)
+	VALUES (6, 'IT');
+    
+-- EDITING A CATEGORY BY AN EMPLOYER
+UPDATE employer_category
+SET category='Tech'
+WHERE employer_id=6;
+
+-- DISPLAYING A CATEGORY BY AN EMPLOYER
+SELECT *
+FROM employer_category
+WHERE employer_id=6;
+
+-- DELETING A CATEGORY BY AN EMPLOYER
+DELETE FROM employer_category
+WHERE employer_id=6;
+-- -----------------------------------------------------------------------
+#iii
+-- POSTING A NEW JOB BY AN EMPLOYER
+
+INSERT INTO posting(employer_id, posting_id, title, description, number_of_openings)
+	VALUES (1, 4, 'Construction worker', 'Infrastructure speaclist', 2);
+    
+-- -----------------------------------------------------------------------
+#iv
+-- Providing a job offer for an employee by an employer.
+
+UPDATE application
+SET offer_time = CURRENT_TIMESTAMP
+WHERE application_id = 1 AND job_seeker_id = 6
+    AND employer_id=1;
+
+-- -----------------------------------------------------------------------
+#v 
+-- Report of a posted job by an employer 
+
+SELECT posting.title, posting.description, date_posted, app.job_seeker_id, status
+FROM posting p, application app
+WHERE  p.posting_id=app.posting_id AND
+p.posting_id=1 AND employer_id=1;
+
+-- -----------------------------------------------------------------------
+#vi 
+-- Report of posted jobs by an employer during a specific period of time (Job
+-- title, date posted, short description of the job up to 50 characters, number
+-- of needed employees to the post, number of applied jobs to the post,
+-- number of accepted offers).
+
+SELECT posting.title, posting.date_posted, posting.description, posting.number_of_openings,
+ COUNT(application.application_id), COUNT(application.status)
+FROM posting p, application app
+WHERE p.posting_id=app.posting_id AND p.employer_id=1 AND
+ date_posted BETWEEN '2019-01-01' AND CURDATE() AND app.status='accepted';
+
+-- -----------------------------------------------------------------------
+#vii 
+-- CREATING AN EMPLOYEE
+
+INSERT INTO users (user_id, email, password)
+    VALUES (12, 'agohier011@booking.com', 'xpnTTJ811');
+    
+INSERT INTO job_seeker (job_seeker_id, first_name, last_name, description, job_seeker_tier)
+	VALUES (12, 'Jen', 'Smith', 'receptionist', 0);
+    
+-- EDITING AN EMPLOYEE INFO
+UPDATE job_seeker
+SET description='Babysitting'
+WHERE job_seeker_id=12;
+
+-- DISPLAYING AN EMPLOYEE
+SELECT *
+FROM job_seeker
+WHERE job_seeker_id=12;
+
+-- DELETING AN EMPLOYEE
+DELETE FROM users
+WHERE user_id=12;
+
+DELETE FROM job_seeker
+WHERE job_seeker_id=12;
+
+-- -----------------------------------------------------------------------
+#viii
+-- Search for a job by an employee.
```

Comment:
A search for a job should go into posting_search (my bad, should have picked better names). It's saying, 'this posting has come up in a search'. Also, we should be testing the given search string on a variety of tables for a match. For example, the search might be an employer category so we should see if we find any matches there.